src/components/atoms/Callout/Callout.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Callout.scss'
+
+const Callout = ({ className, style, content }) => (
+  <article className={`cg-callout ${className}`} style={{ ...style }}>
+    <div>{content}</div>
+  </article>
+)
+
+Callout.defaultProps = {
+  className: '',
+  style: {},
+  content: '',
```

Comment:
remove these default props since it is having any values.

For classnames you can use mods.